### PR TITLE
Mh/version bump/0.1.7 to rc

### PR DESCRIPTION
**Pull request by bleehu**
Whoop! only 13 commits! instead of 130!

### CODE REVIEW

**Comment by Turtlelord26** [comment]
Seems a bit premature. We can't say Melee 2.1 is in until we get improv weapons in too.
Also files say 0.2.0 but PR says 0.1.7

**Comment by bleehu** [comment]
I realised we were due for a major release. I'll hold until melee 2.1 tweaks are done.

File: CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt [hunk 1]
```diff
@@ -101,6 +101,81 @@ payload of your choice.
...
+Contagion Cost: 1 Contagion
+Duration: 10 Rounds
+Range: Payload (use in combination with other Proceeedures)
+
+    This dichloroformaldoxime derivative is known as a "Nettle Agent" which 
+means that only extremely high doses will kill. Some might call that mercy, 
+while those it has been used on know better. Organics exposed to either the 
+liquid (which is a dull amber color with an oily texture) or the vapor almost
+immediately break out in hives, itch all over, serious eye irritation and 
+coughing fits. Extended or concentrated exposure can lead to patches of skin
```
Comment (Reveraine) [inline CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt:116]: Nonblocking 
Conjugation of effects does not match.
Suggest: break out in hives, itch all over, and suffer from serious...fits

**Review by Reveraine — APPROVED** [review]
Awesome!
One non-blocking grammar issue.

**Review by Turtlelord26 — APPROVED** [review]
By File

## Classes

-Doctor gains automatic first two procedures and more on each levelup. Clarifying nonfunctional change. ✔️ 

## Procedures

-Added procedure: Pho-ox. It's on the complex side, but it has numbers and clear effects. Functional, testable. ✔️ 

## Hunter Dog

-Gains Awareness. Testable ✔️ 

## Attachments

-Added Flechette Shells, a shotgun alternate ammo. I want it rephrased at some point for clarity and consistent templating (the bit about range increments is a little jury-rigged, which isn't a criticism at this time, but eventually we'll need to update it). Has numbers and clear effects. Functional, testable ✔️ 

## Readme

inconsistently labelled 2.0 in the readme but 1.7 in the PR title. Not a blocker I'm just making the note.

**Pull request merged**